fms/modules/feedforward.py

```diff
@@ -470,13 +471,31 @@ class ConditionalFeedForward(nn.Module):
         The intermediate size for the expert networks.
     """
 
-    def __init__(self, num_experts: int, dim: int, intermediate_size: int):
+    def __init__(
+        self,
+        num_experts: int,
+        dim: int,
+        intermediate_size: int,
+        use_bias: bool = False,
+        swiglu_limit: float = 7.0,
+    ):
         super().__init__()
         self.num_experts = num_experts
         self.dim = dim
         self.intermediate_size = intermediate_size
         self.w13 = nn.Parameter(torch.empty(num_experts, 2 * intermediate_size, dim))
         self.w2 = nn.Parameter(torch.empty(num_experts, dim, intermediate_size))
+        self.use_bias = use_bias
+        self.swiglu_limit = swiglu_limit
```

Comment:
if this limit is only for gpt-oss, please make the default None and pass in the value fromm the gpt-oss modeling code